cli/testflinger_cli/__init__.py

```diff
@@ -1099,3 +1186,88 @@ def get_job_state(self, job_id):
             # a bad return from get_status()
             logger.debug("Unable to retrieve job state: %s", exc)
         return "unknown"
+
+    def get_agent_state(self, agent_name):
+        """Return the state for the specified agent.
+
+        :param str agent_name: Agent Name
+        :raises SystemExit: Exit with HTTP error code
+        :return Dict: Agent status which includes the state and the queues
+        """
```

Comment:
Can you move the type hints to the function definition?

```suggestion
    def get_agent_state(self, agent_name: str) -> dict:
        """Return the state for the specified agent.

        :param agent_name: Agent Name
        :raises SystemExit: Exit with HTTP error code
        :return: Agent status which includes the state and the queues
        """
```